src/components/modal/CdrModal.vue

```diff
@@ -204,7 +205,62 @@ const addHandlers = () => {
   document.addEventListener('keydown', handleKeyDown);
 };
 
+/**
+ * backgroundContentNodes = nodes to target for aria-hidden
+ * backgroundNodesDiscovered = once true, prevents DOM scraping again
+ */
+const backgroundContentNodes: Array<HTMLElement> = [];
+let backgroundNodesDiscovered = false;
+
+/**
+ * Find the nodes that we should aria show/hide
+ * and store them in local state
+ */
+const findBackgroundContentNodes = () => {
+  // initial selector is complex, breaking it down
+  const notSelectors = [
+    'body > *',
+    ':not(script)',
+    ':not(style)',
+    ':not([aria-hidden=true])',
```

Comment:
Prevents us from accidentally removing `aria-hidden` from something that was already `aria-hidden="true"` on page load.